package.json

```diff
@@ -10,17 +10,18 @@
     },
     "browserslist": "> 0.5%, last 2 versions, not dead",
     "source": "src/index.js",
-    "main": "dist/lib.cjs",
-    "module": "dist/lib.esm.js",
+    "main": "dist/lib/index.js",
+    "module": "dist/lib/index.js",
```

Comment:
We can't ship these changes, but so far I haven't been able to find a way to tell `yalc` to use a different package.json. Ideas?